app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+var cors = require('cors')
+const app     = express();
+app.use(cors())
+require('dotenv').config()
+const host = process.env.NODE_ENV === "production" ? process.env.HOST : 'http://subdomain.jscms.test:6000/';
+const port = process.env.PORT || 6000;
+
+app.get('/', function (req, res) {
```

Comment:
use _req where the variable is not being used. Also add a eslint rule to remove or prefix with _ 'underscore' for an unused variable.